Source/WebCore/platform/graphics/gstreamer/GStreamerCommon.h

```diff
@@ -455,3 +456,4 @@ GstBuffer* gst_buffer_new_memdup(gconstpointer data, gsize size);
 #endif
 
 #endif // USE(GSTREAMER)
+
```

Comment:
Nit: maybe remove this empty line

Reply:
Will do before landing. I think Builder is inserting it. <_<